src/main/java/use_case/list/ListInteractor.java

```diff
@@ -30,7 +30,10 @@ public ListInteractor(UserRepository signupDataAccessInterface,
      */
     @Override
     public void execute(ListInputData listInputData) {
-        final User user = userDataAccessObject.get(userDataAccessObject.getCurrentUsername());
+        final User user = userDataAccessObject.getCurrentUser();
+        if (user == null) {
+            throw new RuntimeException("Not currently logged-in.");
```

Comment:
Edited it to pop up a fail notification instead of throwing an Exception, since if there's a button to refresh the media list manually somehow while a user's not logged in, it'll also reach this place, and that wouldn't be a `RuntimeException` since it's reasonable to expect a user to click on any available buttons they see, I think?